src/lib/jar.js

```diff
@@ -73,6 +73,21 @@ var Jar = {
         var expires = '; expires=' + new Date(new Date().setYear(new Date().getFullYear() + 1)).toUTCString();
         var path = '; path=/';
         document.cookie = obj + expires + path;
+    },
+    getSignedValue: function (cookieName, signedValue, callback) {
```

Comment:
I think leaving it in here is fine for now. While it doesn't really have to do with cookies per se, to me it does seem to be specific to the way _we_ do cookies at this point. So I don't see a reason to split it out, unless we see signing/unsigning as something that we will be doing in other places